### ISSUE TITLE
Key events don't get passed to RPD session

### ISSUE THREAD

**Issue by chilimangoes** [chilimangoes/CasterUIAutomation_POC#2]
In testing this with an RDP connection, the Key events don't seem to fire at all on a remote desktop. The ones in Dragonfly do fire, even if they often have timing/synchronicity issues when modifier keys (SHIFT, CTRL, etc) are used, so a good place to start is looking into the difference between how Dragonfly and the WindowsInput library trigger these keyboard events.

**Comment by Versatilus** [comment]
I haven't even looked at it yet so take this with the appropriate grain of
salt, but I believe the difference is between using scancodes versus
virtual keys. The Windows RDP client passes on scancodes, but it doesn't
try to do anything intelligent with some of the messages Windows sends it.

**Comment by chilimangoes** [comment]
Yeah. Shortly after posting this, I had done a quick Google search on using SendInput with RDP and one of the stackoverflow posts that I ran across mentioned needing to set the scan code on the INPUT struct. And sure enough, WindowsInput doesn't set the scan codes. So I think it should be a fairly straightforward fix, once I get some free time.
